datafusion/core/tests/parquet/schema_adapter.rs

```diff
@@ -293,36 +293,8 @@ async fn test_custom_schema_adapter_and_custom_expression_adapter() {
     ];
     assert_batches_eq!(expected, &batches);
 
-    // Test using a custom schema adapter and no explicit physical expr adapter
-    // This should use the custom schema adapter both for projections and predicate pushdown
```

Comment:
Removing this because it's no longer true. SchemaAdapter is not used for predicate pushdown.